gadget/install/encrypt.go

```diff
@@ -32,11 +32,12 @@ import (
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/osutil/disks"
 	"github.com/snapcore/snapd/secboot"
+	"github.com/snapcore/snapd/secboot/keymgr"
 )
 
 var (
-	secbootFormatEncryptedDevice = secboot.FormatEncryptedDevice
-	secbootAddRecoveryKey        = secboot.AddRecoveryKey
+	secbootFormatEncryptedDevice             = secboot.FormatEncryptedDevice
+	keymgrAddRecoveryKeyToLUKSDeviceUsingKey = keymgr.AddRecoveryKeyToLUKSDeviceUsingKey
```

Comment:
I don't undersand this change,  eventually we will stop adding the recovery key anyway but I would have changed secbootAddRecoveryKey  to call keymgr.AddRecoveryKeyToLUKSDeviceUsingKey instead of doing the change here. As I said I think in the end we want the code to use the keymgr in secboot, see also my PR.

Reply:
I see there's a circular import problem to do what I propose, we need to figure that out, this change is a bit just noise.

Reply:
This has now been reverted